storage/rethinkdb/bootstrap.go

```diff
@@ -0,0 +1,137 @@
+package rethinkdb
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/dancannon/gorethink"
+)
+
+func makeDB(session *gorethink.Session, name string) error {
+	_, err := gorethink.DBCreate(name).RunWrite(session)
+	if err != nil {
+		if strings.Contains(err.Error(), "already exists") {
+			return nil
+		}
+
+		return err
+	}
+
+	resp, err := gorethink.DB(name).Wait().Run(session)
```

Comment:
Non-blocking - the rethink docs say that Wait() is not supported by rethink 2.3 and above - should we include a comment to that effect since rethink 2.3 just came out recently?  (Not sure if/when we'd upgrade)